db.go

```diff
@@ -27,6 +27,14 @@ const magic uint32 = 0xED0CDAED
 // must be synchronzied using the msync(2) syscall.
 const IgnoreNoSync = runtime.GOOS == "openbsd"
 
+// Default values if not set in a DB instance.
+//
+// Do not change while databases are open.
+var (
+	DefaultMaxBatchSize  int = 1000
+	DefaultMaxBatchDelay     = 10 * time.Millisecond
+)
```

Comment:
What about making these constants since the `DB` fields are variable? If you change these to constants then you won't need the "do not change" warning above.